OneWelcomeExampleApp/Interactors/RegisterUserInteractor.swift

```diff
@@ -177,10 +176,12 @@ extension RegisterUserInteractor: RegistrationDelegate {
             handleSecondStepOfTwoStepRegistration()
         case .qrCode:
             registerUserPresenter?.presentQRCodeRegistrationView(registerUserEntity: registerUserEntity)
-        case .stateless, .twoWayStateless:
-            handleStatelessRegistration(challenge)
         default:
-            registrationNotHandled(challenge)
+            if userClient.isStateless {
```

Comment:
How does Example App know about new `.isStateless` property?
![Zrzut ekranu 2025-06-27 o 22 08 03](https://github.com/user-attachments/assets/7d955e22-58c8-4590-a42d-9db1b7c5127e)

Reply:
look at the note

Reply:
but this is not a draft, the PR is finished.